templates/audio-to-text/template.yaml

```diff
@@ -57,6 +57,30 @@ spec:
                   title: Model Server Endpoint
                   type: string
                   description: "The endpoint for an existing model server."
+                includeModelEndpointSecret:
+                  title: Bearer Authentication Required?
+                  type: boolean
+                  default: false
+              dependencies:
+                includeModelEndpointSecret:
+                  allOf:
+                    - if:
+                        properties:
+                          includeModelEndpointSecret:
+                            const: true
+                      then:
+                        properties:
+                          modelEndpointSecretName:
+                            title: Model Endpoint Secret Name
+                            ui:help: Provide the name of the Secret containing your bearer token.
```

Comment:
In a similar vein to @maysunfaisal's comment, since the secret must be in the same namespace as the deployment namespace, it might be good to highlight that requirement here.

Reply:
+1 (provided more context [here](https://github.com/redhat-ai-dev/ai-lab-template/pull/40#discussion_r1803619385))